.github/workflows/quarkus-master-cron.yaml

```diff
@@ -190,7 +190,7 @@ jobs:
       - name: Report build status
         id: report
         run: |
-          ./mvnw verify -N -B -ntp -Pbuild-notification -Dstatus=verify -Dtoken=${{ secrets.GITHUB_TOKEN }} -DbuildId=$(cat ~/build-data/build-id.txt) -Drepo=${GITHUB_REPOSITORY} -Dbranch=quarkus-master
+          ./mvnw verify -N -B -ntp -Pbuild-notification -Dstatus=verify -Dtoken=${{ secrets.GITHUB_TOKEN }} -DbuildId=$(cat ~/build-data/build-id.txt) -Drepo=${GITHUB_REPOSITORY} -Dbranch=quarkus-master -Dbranch-commit=$(cat ~/build-data/master-sha.txt) -Dbranch-commit=$(cat ~/build-data/master-sha.txt)
```

Comment:
Looks like you added a duplicate argument on this one.

Reply:
Good catch, thanks!